Description: Remove argparse from requirements.txt
 Note: this is already fixed upstream.
Author: Thomas Goirand <zigo@debian.org>
Forwarded: not-needed
Last-Update: 2016-03-10

--- python-keystoneauth1-2.1.0.orig/requirements.txt
+++ python-keystoneauth1-2.1.0/requirements.txt
@@ -2,7 +2,6 @@
 # of appearance. Changing the order has an impact on the overall integration
 # process, which may cause wedges in the gate later.
 
-argparse
 iso8601>=0.1.9
 requests>=2.8.1
 six>=1.9.0
